src/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  margin: 0;
+}
+
+.parent-block {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.child-block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  color: white;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
+@media (min-width: 900px) {
+  .parent-block {
+    height: 600px;
```

Comment:
interesting solution 👍, but usually if you work with the width you would use flex-basis.